app/models/book.rb

```diff
@@ -0,0 +1,2 @@
+class Book < ApplicationRecord
```

Comment:
Do not forgot to validate your model here also. (for example if field is required or no)

Reply:
I introduced those validations on the book_testing branch. I could go back and include them here too.

Reply:
Yes I would prefer :)